CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.5.0] - 2017-12-12
+
+### Added
+- beetmoverscript support for Devedition releases
```

Comment:
We seem to also have added `.beet` support in general, no?
Also, `get_product_name`, `NORMALIZED_BALROG_PLATFORMS`.